addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -97,12 +97,12 @@ var Wysiwyg = Widget.extend({
             },
             snippetMenuElement: $mainSidebar[0],
             snippetManipulators: $snippetManipulators[0],
-            customCommands: { ...{
+            customCommands: Object.assign({
```

Comment:
Why ? Genuinely wondering.

Reply:
The spread syntax for object literals was introduced in ES2018. Odoo is currently stuck in ES2017 so everywhere we use that syntax we should replace it with `Object.assign`.